src/main/fc/rc_controls.c

```diff
@@ -142,23 +154,23 @@ void processRcStickPositions(throttleStatus_e throttleStatus)
             stTmp |= 0x40;  // check for MAX
     }
     if (stTmp == rcSticks) {
-        if (rcDelayCommand < 250)
-            rcDelayCommand++;
-    } else
-        rcDelayCommand = 0;
+        if (rcDelayMs < INT16_MAX)
```

Comment:
This check does not effectively prevent a range overflow. It should be `if (rcDelayMs <= INT16_MAX - (getTaskDeltaTime(TASK_SELF) / 1000)`.